### PR TITLE
New Wordlist - Software

### PR DESCRIPTION
**Pull request by zacharysarette**
Useful words for talking about kinds of software.

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -113,6 +115,15 @@ Specification document | 仕様書 | しようしょ |
 Feature | 機能 | きのう |
 
 
+### Software
+
+English | Japanese | Kana | Notes
+--------|----------|------|------
+Word Processor | ワープロ |
+Word Processing Software | ワープロソフト |
+Spread Sheet | 表計算 | ひょうけいさん |
```
Comment (ronkorving) [inline ReadMe.md]: Just one tiny suggestion. I believe "spreadsheet" is one word.

```suggestion
Spreadsheet | 表計算 | ひょうけいさん |
```

**Comment by ronkorving** [comment]
Thanks for the contribution, @zacharysarette !
I left one small suggestion, please have a look.

File: ReadMe.md (hunk 1 above)
Comment (zacharysarette) [reply ReadMe.md]: Agreed.

**Comment by ronkorving** [comment]
Thanks! :)

**Pull request merged**